PyDSS/dssInstance.py

```diff
@@ -457,7 +458,8 @@ def RunSimulation(self, project, scenario, MC_scenario_number=None):
         self._Logger.info("Set OpenDSS convergence to %s", dss.Solution.Convergence())
         self._Logger.info('Max convergence error count {}.'.format(self._maxConvergenceErrorCount))
         self._Logger.info("initializing store")
-        self.ResultContainer.InitializeDataStore(project.hdf_store, Steps, MC_scenario_number)
+        # make space for Steps*5 to allow for co-iterations
+        self.ResultContainer.InitializeDataStore(project.hdf_store, Steps * 10, MC_scenario_number)
```

Comment:
@nadiavp @AadilLatif Can the number of steps be determined? I would prefer that this value be exact if that is possible. I had been under the assumption that we would store one value per time step. Can we skip co-iterations where the "federation" does not move forward?

Reply:
If HELICS then this needs to steps * max_coiterations.